src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/MarshallerShape.cs

```diff
@@ -411,12 +411,30 @@ public record MarshallerMethods
             public IMethodSymbol? ToUnmanaged { get; init; }
             public IMethodSymbol? Free { get; init; }
             public IMethodSymbol? OnInvoked { get; init; }
+            public IMethodSymbol? StatefulGetPinnableReference { get; init; }
```

Comment:
I'm assuming we will have a `StaticGetPinnableReference` at some point?

Reply:
Yeah I'll probably add it very soon as part of the analyzer/codefix work.